src/js/sources-aggregate.js

```diff
@@ -145,12 +145,18 @@ export const displayedMessages = derived(dispDepends, dispTransform);
 export const captionText = readable(defaultCaption, set => {
   let text = defaultCaption;
   return displayedMessages.subscribe($msgs => {
-    if ($msgs[$msgs.length - 1]?.text != null && $msgs[$msgs.length - 1]?.text !== text) {
+    if ($msgs[$msgs.length - 1]?.text != null &&
+      $msgs[$msgs.length - 1]?.text !== text &&
```

Comment:
`$msgs[$msgs.length - 1]`
what if there's a message between the duplicated message 1 and 2? i feel like this is a patch that isn't fixing the fundamental problem

Reply:
![image](https://user-images.githubusercontent.com/38841491/163455759-d8c4ce6e-81d1-4f60-8077-a42df61bbc01.png)

Reply:
This is the read aloud duplication, it is only supposed to check against the last message.